### ISSUE TITLE
Video/Audio Caption

### ISSUE THREAD

**Issue by sareh** [bbc/simorgh#866]
**Is your feature request related to a problem? Please describe.**
We should display a caption if the media block has caption data.

**Describe the solution you'd like**
We should have this stucture:
```
<figure>
    <video />
    <figcaption>caption here</figcaption>
</figure>
```

Note that we should ensure this works for the AMP variant as well as for canonical.

- The approach should reuse/adapt the existing image caption code as far as possible to reduce maintenance overhead.
- We should ensure that italics are still not rendered in video captions for Arabic-script sites (as with image captions).

We should have a storybook item showing the Video block with Caption for AMP and Canonical variants. Can use the Image examples in Psammead as a reference of how to have AMP and Canonical variants: https://bbc.github.io/psammead/?path=/story/image-ampimg--landscape-image
https://github.com/bbc/psammead/tree/latest/packages/components/psammead-image/src

The caption should be the same one as that used for the image, except this has visually hidden text that says 'Video caption,' or 'Audio caption,' rather than 'Image caption,'. 

<img width="841" alt="screen shot of video in article" src="https://user-images.githubusercontent.com/3028997/56123990-89d4a800-5f6d-11e9-928e-285ce8ea500b.png">

https://app.zeplin.io/project/5baa564c05c2eba0d2b89568/screen/5c596fa94e189735593e60e8

Note: captions can display rich text formatting.

**Describe alternatives you've considered**
N/A

**Testing notes**

This should only have unit tests for the component, since it won't be integrated into the article yet.
The caption should meet all accessibility criteria that exist for the image caption. 

This will require an `a11y` swarm once the logic is ready/available.

We can test using fixture data taken from this Test article. This is because this article has a variety of video blocks: https://www.test.bbc.co.uk/news/articles/c2nn1prnl39o.json 

Storybook should have an example of a 'video component' without a caption and one with a caption. Note: the 'video component' renders just text values at the moment - we have another PR to add in an actual video element.

Captions on RTL services (Arabic, Persian, Pashto and Urdu) can't use italics, so audio/video captions should not appear in italics.

**Additional context**
**This issue does not have a video rendering as a prerequisite.** It should be sufficient to just have a figure and a figcaption element, without the video actually rendering. 

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)

**Comment by jimjohnsonrollings** [comment]
This was reverted with the other AV work, but remains a requirement.

**Comment by 12** [comment]
Whoever picks this up can use a similar implementation to what had been done previously.

I removed the caption block as its implementation required integrating the nested grid logic which was quite untidy initially. I imagine some of this logic will need reintroducing. You can browse the previous implementation here e08442124ee64c8e1e2421edc5b045c42a3bceec, it was relatively simple:

https://github.com/bbc/simorgh/blob/e08442124ee64c8e1e2421edc5b045c42a3bceec/src/app/containers/MediaPlayer/index.jsx#L57
https://github.com/bbc/simorgh/blob/e08442124ee64c8e1e2421edc5b045c42a3bceec/src/app/containers/MediaPlayer/index.jsx#L109
